tests/test_workers/test_tasks/test_utils.py

```diff
@@ -864,9 +864,9 @@ def test_prepare_request_for_build_merge_index_img(mock_gia, mock_gri, mock_giii
 
     target_index_info = {
         'resolved_from_index': 'some_other_image@sha256',
-        'ocp_version': 'v4.6',
+        'ocp_version': 'v4.9',
         'arches': {'amd64'},
-        'resolved_distribution_scope': 'stage',
+        'resolved_distribution_scope': 'prod',
```

Comment:
Why is this changed?

Reply:
To make sure the tests break if something changes in `prepare_request_for_build`